src/fetchMissing.js

```diff
@@ -0,0 +1,43 @@
+const fetch = require('node-fetch')
+const utils = require('web3-utils')
+
+async function getLogo(xdaiAddr) {
+  const response = await fetch(`https://blockscout.com/poa/xdai/tokens/${xdaiAddr}/token-transfers`)
+  const text = await response.text()
+  const mainnetAddr = text.match(/https:\/\/etherscan.io\/token\/(0x[a-fA-F0-9]{40})/)[1]
+  const formattedAddr = utils.toChecksumAddress(mainnetAddr)
+  return `https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/${formattedAddr}/logo.png`
+}
+
+async function fetchMissing() {
+  const response = await fetch(
+    'https://blockscout.com/poa/xdai/api/?module=account&action=tokentx&address=0xf6A78083ca3e2a662D6dd1703c939c8aCE2e268d'
+  )
+  const data = await response.json()
+  const bridgedTokens = [...new Set(data.result.map(x => x.contractAddress.toLowerCase()))]
+
+  const response2 = await fetch(
+    'https://raw.githubusercontent.com/1Hive-tulip/default-token-list/master/uniswap-xdai.tokenlist.json'
```

Comment:
We will soon be using IPFS to host the list, but this is just the JSON built from `npm run build` so maybe we can use 'build/honeyswap-default.tokenlist.json' for this step.

This is not in the scope of this PR, but I think it would be nice to add a command `npm run update-list` that builds first and then uses this script to update xdai.json rather than logging it.